RELEASE_CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-### [v2.1.0](https://github.com/provenance-io/provwasm/tree/v2.1.0)
+### [v2.2.0](https://github.com/provenance-io/provwasm/tree/v2.2.0)
```

Comment:
Same as above. Should the optimizer update be included?

Reply:
fixed